packages/opentelemetry-sdk-trace-base/src/Tracer.ts

```diff
@@ -120,6 +123,18 @@ export class Tracer implements api.Tracer {
       return api.trace.wrapSpanContext(spanContext);
     }
 
+    let clock: AnchoredClock | undefined;
+    if (parentSpan) {
+      clock = (parentSpan as any)['_clock'];
+    }
+
+    if (!clock) {
+      clock = new AnchoredClock(Date, otperformance);
```

Comment:
Do we have any numbers on performance impact by using `Date` instead of hrtime?
If impact is significant we might consider to allow users to opt out if they know their app runs in an environment which has a stable hr timer.

Reply:
I don't. We could make a microbenchmark which shows the speed difference of the actual functions, but impact on a real world application is likely to be significantly different.

Reply:
>if they know their app runs in an environment which has a stable hr timer.

I think explaining to users what is meant by a stable hr timer is going to be difficult. just because your computer doesn't sleep doesn't mean you don't drift over time for example.

Reply:
I can't believe when I posted this I didn't notice that Date is actually the faster one. I just tried to recreate this and got the same result again. So it seems the performance advantage of perf is not only not there but is actually a very slight disadvantage. The difference is still so small that I don't think it will matter though